test/mock/fake_clientset.go

```diff
@@ -44,7 +44,7 @@ func (f *FakeClientset) Clientset() *kubernetes.Clientset {
 func (f *FakeClientset) roundTripperFn(req *http.Request) (*http.Response, error) {
 	switch method := req.Method; {
 	case method == "GET":
-		body := ioutil.NopCloser(bytes.NewReader([]byte(runtime.EncodeOrDie(f.codec, f.pod))))
+		body := io.NopCloser(bytes.NewReader([]byte(runtime.EncodeOrDie(f.codec, f.pod))))
```

Comment:
This fixes #133

Reply:
Yep, I should have separated it. But was too lazy for it.